### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by dotperinch**

### CODE REVIEW

**Comment by dotperinch** [comment]
Now!

File: README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
 # test
 # foo
-# bar
\ No newline at end of file
+
```
Comment (mw76) [inline README.md]: Is this empty line necessary?

Comment (dotperinch) [reply README.md]: Always ... leave an empty line ,-).

File: README.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
...
+# bar
+# baz
+
```
Comment (mw76) [inline README.md]: No empty lines at the end of files, please!

**Pull request merged**